index.html

```diff
@@ -56,7 +63,7 @@ <h2 class="bold">NEXT RACING EVENT - Sun, 29 May, 14:00</h2>
         </div>
     </header>
     <main class="container main text-light">
-        <h2 class="center mg-1"> Main Program </h2>
+        <h2 class="center mg-1 main-header"> Main Program </h2>
```

Comment:
- [ ] [OPTIONAL] Please consider changing the background to the one that was used in the original design or a closer one with the same color
- [ ] [OPTIONAL] Kindly use the same colors as the original design for the program items images and add a hover effect that adds a border to the program items (squares)
- [ ] Kindly center the element and use the propper ```padding```/```margin``` so your design is identical to the given one

|Your design|CC design|
|-|-|
|![Screenshot_20220528_055610](https://user-images.githubusercontent.com/25227022/170810792-4949e796-ef63-429b-8e97-ed9762c6d911.png)|![Screenshot_20220528_060048](https://user-images.githubusercontent.com/25227022/170810805-23dd30d7-4031-4ebc-88a6-056852ea5032.png)|